MODULE.bazel

```diff
@@ -12,12 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO(fchern): automate version string alignment with setup.py
-VERSION = "0.6.0"
-
 module(
     name = "array_record",
-    version = VERSION,
+    version = "0.7.3",
     repo_name = "com_google_array_record",
 )
 
```

Comment:
Let's use actual `bazel_dep(name = "protobuf", version = ...)` here and enforce errors if different is picked.